refinery_rules.md

```diff
@@ -1,3 +1,5 @@
+<!-- Do not edit manually. This is an AUTO-GENERATED file. -->
```

Comment:
This is not auto-generated. The content is auto-generated in Refinery, then copy-pasted here. See #2545 for an example of the last update made.